### PR TITLE
datamodal typecompenents langauage update

### PR DESCRIPTION
**Pull request by salihkiraz**
-  DataModule link options
image

image

- Modal Cache problem fix
- Data table list header value options
image

### CODE REVIEW

File: src/views/default/type_components/datamodal/component.blade.php [hunk 1]
```diff
@@ -22,13 +26,10 @@
...
-	var url_{{$name}} = "{{CRUDBooster::mainpath('modal-data')}}?table={{$form['datamodal_table']}}&columns=id,{{$form['datamodal_columns']}}&name_column={{$name}}&where={{urlencode($form['datamodal_where'])}}&select_to={{ urlencode($form['datamodal_select_to']) }}";
-	var url_is_setted_{{$name}} = false;
+	var url_{{$name}} = "{{CRUDBooster::mainpath('modal-data')}}?table={{$form['datamodal_table']}}&columns=id,{{$form['datamodal_columns']}}&name_column={{$name}}&where={{urlencode($form['datamodal_where'])}}&select_to={{ urlencode($form['datamodal_select_to']) }}&columns_name_alies={{ urlencode($form['datamodal_columns_alias_name']) }}";
+
 	function showModal{{$name}}() {
-		if(url_is_setted_{{$name}} == false) {								    			
-    		url_is_setted_{{$name}} = true;
-    		$('#iframe-modal-{{$name}}').attr('src',url_{{$name}});
-		}
+    $('#iframe-modal-{{$name}}').attr('src',url_{{$name}});
```
Comment (fherryfherry) [inline src/views/default/type_components/datamodal/component.blade.php]: This is for remove datamodal cache right ?

File: src/views/default/type_components/datamodal/component.blade.php [hunk 2]
```diff
@@ -6,13 +6,17 @@
...
 	?>	
 	
 	<div id='{{$name}}' class="input-group">
 	  <input type="hidden" name="{{$name}}" class="input-id" value="{{$value}}">
       <input type="text" class="form-control input-label {{$form['required']?"required":""}}" value="{{$datamodal_value}}" readonly>
       <span class="input-group-btn">
-        <button class="btn btn-primary" onclick="showModal{{$name}}()" type="button"><i class='fa fa-search'></i> Browse Data</button>
+        <button class="btn btn-primary" onclick="showModal{{$name}}()" type="button"><i class='fa fa-search'></i> {{trans('crudbooster.datamodal_browse_data')}}</button>
+			<?php if(strlen($form['datamodal_module_name'])>1){ ?>	
+        <a class="btn btn-info"  href="../../{{$form['datamodal_module_name']}}" target="_blank"><i class='fa fa-edit'></i> {{$form['label']}}</a>
```
Comment (fherryfherry) [inline src/views/default/type_components/datamodal/component.blade.php]: I don't really understand about this line, can you tell me? what is it? and why using a relative path? thx

File: src/views/default/type_components/datamodal/component.blade.php (hunk 1 above)
Comment (salihkiraz) [inline src/views/default/type_components/datamodal/component.blade.php]: yes cache problem fix.

File: src/views/default/type_components/datamodal/component.blade.php (hunk 2 above)
Comment (salihkiraz) [inline src/views/default/type_components/datamodal/component.blade.php]: I did it wrong.
Correct code:
href="{{CRUDBooster::adminPath()}}/{{$form['datamodal_module_name']}}"

Comment (fherryfherry) [inline src/views/default/type_components/datamodal/component.blade.php]: You mean is when user click that button, will be redirected to its module?

Comment (salihkiraz) [inline src/views/default/type_components/datamodal/component.blade.php]: yes. tested

Comment (fherryfherry) [inline src/views/default/type_components/datamodal/component.blade.php]: Ok, I think **datamodal_module_slug** is better ??

Comment (salihkiraz) [inline src/views/default/type_components/datamodal/component.blade.php]: ok.

Comment (fherryfherry) [inline src/views/default/type_components/datamodal/component.blade.php]: I've no confirm yet, have you solve this ?

Comment (fherryfherry) [inline src/views/default/type_components/datamodal/component.blade.php]: *datamodal_module_slug* sorry, it better *datamodal_module_path*

**Pull request merged**